MinecraftClient/config/ChatBots/discordWebhook.cs

```diff
@@ -0,0 +1,369 @@
+//MCCScript 1.0
+//using System.Collections.Specialized;
+
+MCC.LoadBot(new DiscordWebhook());
+
+//MCCScript Extensions
+class SkinAPI
+{
+    private Dictionary<string, string> skinModes = new Dictionary<string, string>();
+    private string currentSkinMode;
+    private int size;
+    private int scale;
+    private bool overlay;
+    private string fallbackSkin;
+
+    public SkinAPI(string newCurrentSkinMode = "flatFace", string newFallbackSkin = "MHF_Steve", int newSize = 512, int newScale = 10, bool newOverlay = true)
+    {
+        skinModes.Add("flatFace", "https://crafatar.com/avatars/{0}");
+        skinModes.Add("cubeHead", "https://crafatar.com/renders/head/{0}");
+        skinModes.Add("fullSkin", "https://crafatar.com/renders/body/{0}");
+
+        currentSkinMode = newCurrentSkinMode;
+        fallbackSkin = newFallbackSkin;
+        size = newSize;
+        scale = newScale;
+        overlay = newOverlay;
+    }
+
+    // Always get a an Answer to a valid name!
+    public string getUUIDFromMojang(string name)
+    {
+        WebClient wc = new WebClient();
+        return Json.ParseJson(wc.DownloadString("https://api.mojang.com/users/profiles/minecraft/" + name)).Properties["id"].StringValue;
+    }
+
+    // May fail on non premium Servers!
+    public string getUUIDFromPlayerList(string name, Dictionary<string, string> playerList)
+    {
+        return playerList.FirstOrDefault(x => x.Value == name).Key.Replace("-", "");
+    }
+
+    public string getSkinURL(string UUID)
+    {
+        string parameters = string.Join("&", "size=" + size, "scale=" + scale, "default=" + fallbackSkin, (overlay ? "overlay" : ""));
+        return string.Format(skinModes[currentSkinMode], UUID + "?" + parameters);
+    }
+
+    public void setSize(int newSize) { if (newSize <= 512) { size = newSize; } }
```

Comment:
You may use setter/getter (Auto-Property) here.
```cs
public int Size 
{ 
    set { if (value <= 512) size = value; } 
    get { return size; }
}
```

Reply:
I'd even say
```C#
set
{
    size = value;
    if (size > 512)
        size = 512;
}
```
So that size changes to max if provided value is too big.